src/components/experimental/Button/Button.tsx

```diff
@@ -62,6 +62,10 @@ const ButtonStyled = styled(BaseButton)<{ $emphasis: Emphasis }>`
 
     cursor: pointer;
 
+    &[data-disabled] {
+        cursor: not-allowed;
```

Comment:
I'd do "default" cursor, not-allowed is too dramatic :D

Reply:
Hehe I don't have a strong opinion on the subject, but since the [best practices](https://webreference.com/css/properties/cursor) suggests
> Use the "not-allowed" cursor for disabled elements to indicate that they are not clickable.

I thought it's better to keep it this way, considering we don't have any design specs for how it should look like. With the default pointer it kinda looks to me like the disabled element isn't there and that's not the best user experience imho

Reply:
I will keep it for now @alatielle and mark this as resolved, there's a chance the designers agree with you, promise to patch it super quick in that case 🫰